src/main/java/com/radixdlt/crypto/Signatures.java

```diff
@@ -0,0 +1,66 @@
+package com.radixdlt.crypto;
+
+import java.util.Map;
+
+public interface Signatures {
+
+    /**
+     * Returns the {@link SignatureScheme} used to produce the signatures of this collection.
+     * @return Returns the {@link SignatureScheme} used to produce the signatures of this collection.
+     */
+    SignatureScheme signatureScheme();
+
+    /**
+     * Returns the type of {@link Signature} produced by the {@code signatureScheme}.
+     * @return Returns the type of {@link Signature} produced by the {@code signatureScheme}.
+     */
+    Class<? extends Signature> signatureType();
```

Comment:
@flotothemoon yes maybe makes more sense to change this to `SignatureType signatureType()` where `SignatureType` is an enum, that might still conform to the trivial interface `Signature`